lib/simp/cli/commands/bootstrap.rb

```diff
@@ -25,6 +23,11 @@ def initialize
     @track = true
     @unsafe = false
     @verbose = false
+
+    @env_helper = Simp::Cli::Config::SimpPuppetEnvHelper.new(Simp::Cli::BOOTSTRAP_PUPPET_ENV)
```

Comment:
The use of this object is supposed to be temporary until `Simp::Cli::Environment::OmniEnvController` supports a status method.  So, I would rather keep this code localized as it was before (`check_for_simp_environment()`) and use Simp::Cli::Utils::is_pe? directly. (`Simp::Cli::Config::SimpPuppetEnvHelper` is designed for `simp config` not all commands).

Reply:
Actually, would rather see `Simp::Cli::Utils::is_pe?` called here after the loading of the facts for the configured bootstrap environment.  Don't want the facts from all environments, as that may cause problems down the line.

Reply:
I was trying to change as little logic as possible to make this work so I didn't want to fuss about with small changes that would modify the current behavior.

Each call to `load_custom_facts` clears the entire fact cache so there shouldn't be an issue with things being reloaded.

Reply:
@lnemsick-simp In terms of the `@env_helper` code, I needed it in the line below and thought that having two locations with the full call seemed unnecessary. I'll move the comment up with the main call though.